README.md

```diff
@@ -667,3 +668,8 @@ This will return a response in the following format
 | 048 | Children's Centre Linked Site|
 | 056 | Institution funded by other government department|
 | 057 | Academy secure 16 to 19|
+
+
+
+## Postman collection
+We do have a Postman workspace with some sample requests, which is available upon request on Slack. 
```

Comment:
As suggested, can you please add the name of the slack channel - dfe_sign-in please.

Reply:
That's done now.